commons/src/main/java/org/eclipse/kapua/commons/configuration/AbstractKapuaConfigurableService.java

```diff
@@ -129,7 +129,7 @@ private void validateConfigurations(String pid, KapuaTocd ocd, Map<String, Objec
                 }
             }
 
-            String logicValidationResult = validateNewConfigValuesCoherence(ocd, updatedProps, scopeId);
+            String logicValidationResult = validateNewConfigValuesCoherence(ocd, updatedProps, scopeId, parentId);
```

Comment:
Well, some class inheriting `AbstractKapuaConfigurableService` could provide its own validation logic in an overriding method, so it's not guaranteed that such call will always return an empty string. I thought that checking for some service-specific logic first was a good idea, to avoid reading configurations from the database when we already know that the custom logic will fail.